x-pack/plugins/fleet/server/routes/data_streams/handlers.ts

```diff
@@ -172,6 +173,19 @@ export const getListHandler: RequestHandler = async (context, request, response)
                 size: 1,
               },
             },
+            serviceName: {
+              terms: {
+                field: 'service.name',
+                size: 2,
+              },
+            },
+            environment: {
+              terms: {
+                field: 'service.environment',
+                size: 1,
```

Comment:
probably want to set size to 2 here as well (and link to ENVIRONMENT_ALL if there are multiple environments).